tapdb/sqlc/migrations/000010_universe_stats.down.sql

```diff
@@ -0,0 +1,2 @@
+-- This file is empty on purpose. There is nothing to roll back for this
```

Comment:
So re stats for transfers vs assets, the intent was that we started to use the other enum type to differentiate syncing of transfer vs issuance proof: https://github.com/lightninglabs/taproot-assets/blob/81eb8d72e75dd567ba4e2ff3f15a1226a2d20fdc/universe/base.go#L535-L543

Reply:
We also store the proof type here as well: https://github.com/lightninglabs/taproot-assets/blob/main/tapdb/universe_stats.go#L143C31-L143C31

Reply:
We're sorta just not using the total "syncs" count anymore, but we can still obtain stats about the number of transfers.

Reply:
Hmm, not 100% sure what you mean. The proof type here https://github.com/lightninglabs/taproot-assets/blob/main/tapdb/universe_stats.go#L143C31-L143C31 refers to the [`roots.proof_type`](https://github.com/lightninglabs/taproot-assets/blob/81eb8d72/tapdb/sqlc/queries/universe.sql#L127) of the `InsertNewSyncEvent` query.

But you're saying we should start distinguishing between `NEW_PROOF` (for transfers?) and `NEW_ROOT` (for issuances)?

Does sound like a fix for another PR though. Also, inserting a row per event does look like a scaling issue. Maybe an upsert on a bucketized time stamp, so we'd only have one entry per 5 minutes and event type?